torch/_tensor_docs.py

```diff
@@ -769,6 +769,13 @@ def add_docstr_all(method, docstr):
 In-place version of :meth:`~Tensor.bitwise_xor`
 """)
 
+add_docstr_all('broadcast_to',
+               r"""
+broadcast_to(input, size) -> Tensor
```

Comment:
"broadcast_to(input, size)" -> "broadcast_to(shape)"

The input is implied for methods, and the word "shape" is consistent with NumPy and other PyTorch functions.

Reply:
Thanks so much for pointing this out, the method signature is updated accordingly in the latest PR.